SEED-platform/seed
# BuildingSync Version 1.0 Support, Remove Unused Tables, Add ULID and Footprints

#### What's this PR do?
Updated buildingsync parser to v1.0.0
Updated enumerations.json file for v1.0.0 and added schema_version and schema_type to measures model so we can track different schemas in the future.
#### How should this be manually tested?
Upload a v1.0.0 xml (should work)
Upload a non v1.0.0 xml (should fail)

#1832, #1831
